docs/BREAKINGCHANGES.md

```diff
@@ -195,16 +171,16 @@ TODO
 
 
 ### Remove ClrVersion property from $PSVersionTable [4027](https://github.com/PowerShell/PowerShell/issues/4027)
-TODO
+The ClrVersion property of `$PSVersionTable` is not useful with CoreCLR,  end users should not be using that value to determine compatibility.
 
 
 ### Command line arguments with a dollar sign [4024](https://github.com/PowerShell/PowerShell/issues/4024)
 TODO
 
 
 ### Change positional parameter for powershell.exe from -Command to -File [4019](https://github.com/PowerShell/PowerShell/issues/4019)
-TODO
-
+This fixes the usage of `#!` in PowerShell scripts that are being executed from non-PowerShell shells on non-Windows platforms.
```

Comment:
Enable shebang use of PowerShell on non-Windows platforms.  This means on Unix based systems,
you can make a script executable which would invoke PowerShell automatically rather than explicitly
invoking pwsh.